app/emails/email.py

```diff
@@ -0,0 +1,16 @@
+from flask_mail import Message
+from app import mail
+from flask import render_template, current_app
+
+def send_email(subject, sender, recipients, text_body, html_body):
+    msg = Message(subject, sender=sender, recipients=recipients)
+    msg.body = text_body
+    msg.html = html_body
+    mail.send(msg)
+
+def send_password_reset_email(user):
+    token = user.get_reset_password_token()
+    send_email('[Microblog] Reset Your Password',
+               sender='jerry.aragon@student.csulb.edu', recipients=[user.email],
+               text_body=render_template('email/reset_password.txt', user=user, token=token),
+               html_body=render_template('email/reset_password.html',user=user, token=token))
```

Comment:
I'm not sure if you already made an issue for this yet, but the sending email function should be called in a separate thread. Just a suggestion if for future sprints. The tutorial does cover how to setting this up.

Reply:
OK sounds good, I might leave this for the next sprint, ill just add the issue to the backlog. I saw the tutorial mentioning putting this in a thread, but I didn't bother to do it since it wasn't a huge requirement.